app/src/lib/calculateMetadata/epoca.ts

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from 'luxon'
+
+export const epoca = (dateobs: Date): number => {
+    const t = DateTime.fromJSDate(dateobs) // Convierte el objeto Date a un DateTime de Luxon
+    const year = t.year + (t.ordinal - 1) / 365
+    return parseFloat(year.toFixed(1)) // Devuelve el año con una decimal
+}
```

Comment:
Se podría usar `date-fns` en vez de agregar `luxon` como dependencia.

```suggestion
import { getDayOfYear, getDaysInYear } from 'date-fns'

export const epoca = (dateobs: Date): number => {
  const days = getDaysInYear(dateobs) // 365 o 366
  const year = dateobs.getFullYear() + (getDayOfYear(dateobs) - 1) / days
  return parseFloat(year.toFixed(1)) // Devuelve el año con una decimal
}
```